synapse/lib/cell.py

```diff
@@ -303,6 +320,115 @@ class Cell(s_base.Base, s_telepath.Aware):
             await user.setPasswd(passwd)
             self.insecure = False
 
+        await self._initCellHttp()
+
+        async def fini():
+            [await s.fini() for s in self.sessions.values()]
+
+        self.onfini(fini)
+
+    def _getSessInfo(self, iden):
+        return self.sessstor.gen(iden)
+
+    async def genHttpSess(self, iden):
+
+        # TODO age out http sessions
```

Comment:
I don't see anything that just closes the session when the user disconnects either. Did I miss it?

Reply:
In a multi-connection protocol like HTTP, there isn't a single connection you can use to limit session boundaries.  Age ( and most specifically time since last request ) is the only reasonable approach.  That'll be coming in a subsequent delta.